JSDemos/Demos/FileUploader/FileSelection/React/App.tsx

```diff
@@ -6,6 +6,7 @@ import notify from 'devextreme/ui/notify';
 
 const firstNameLabel = { 'aria-label': 'First Name' };
 const lastNameLabel = { 'aria-label': 'Last Name' };
+const fileUploaderLabel = { 'aria-label': 'Select photo' };
```

Comment:
Should we change 'Select photo' with 'Select Photo'? Usually when using inputAttr, the text of the attribute is written in capital letter. At least there are many examples of such cases in the project.

Reply:
[a41ca504a5fd83fef8b291fb96c8c7fa70d01400]